Makefile

```diff
@@ -439,6 +439,16 @@ else
 	scripts/push-to-kind.sh -i ${VLOGGER_IMG}
 endif
 
+# Vertica client proxy is a pre-built image that we don't build. For this
+# reason, pushing this image will just put it in kind and never to docker.
+.PHONY: docker-push-vproxy
+docker-push-vproxy:  ## Push Vertica client proxy image to kind
+ifneq ($(VPROXY_IMG), <not-set>)
```

Comment:
VPROXY_IMG has a default value, will this condition work?

Reply:
You are right. Just remove the condition.